### PR TITLE
Update mockery version

### PR DESCRIPTION
**Pull request by alex-semenyuk**
Fix issue on `make mocks`
```
internal error: package "context" without types was imported from "github.com/hyperledger/firefly-transaction-manager/pkg/fftm"
make: *** [mocks--Manager] Error 1
```

Explicit define version instead of latest for mockery as recommended  https://vektra.github.io/mockery/latest/installation/

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/hyperledger/firefly-transaction-manager
 
-go 1.21
```
Comment (Chengxuan) [inline go.mod:3]: We usually do golang version update across Firefly repos and don't normally pin to a specific patch version. e.g., it should be `go 1.22` in this example

@EnriqueL8 is there a plan for the minor version 22 yet?

Comment (EnriqueL8) [reply go.mod:3]: I haven't set a plan yet for it, will keep that in mind - @alex-semenyuk is it strictly needed for this change?

Comment (EnriqueL8) [reply go.mod:3]: @alex-semenyuk ^^^^

Comment (alex-semenyuk) [reply go.mod]: @EnriqueL8 sorry for delay, yes, we can move without version update, so updated PR
Btw noticed that there is intermittently failed test https://github.com/hyperledger/firefly-transaction-manager/issues/127

**Review by Chengxuan — APPROVED** [review]

**Pull request merged**